drivers/wdm/audio/backpln/portcls/irpstream.cpp

```diff
@@ -496,7 +534,8 @@ NTAPI
 CIrpQueue::NumData()
 {
     // returns the amount of audio stream data available
-    return m_NumDataAvailable;
+    UNIMPLEMENTED;
+    return 0;
```

Comment:
It's not possible now to know this data? Or is NumData() going to be deprecated?

Reply:
The value can be computed on demand pretty easily, I can code that up now if you want. I just wanted to get the ball rolling with this PR before I just kept messing around with it forever.